### PR TITLE
XRT-2769: fix profile location in server examples

### PR DESCRIPTION
**Pull request by JonathanWhiteUK**
Just moved profile into state folder

### CODE REVIEW

File: Servers/opc-ua/federated/README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Now we have the Browser working we can move on to setting up XRT OPC UA Server f
 2. **Set Environment Variables**:
 
     ```shell
-    . ./../set_env.sh
+    . ../../../set_env.sh
```
Comment (AIM289) [inline Servers/opc-ua/federated/README.md]: I think the script this is pointing to is named set_env_vars.sh

File: Servers/opc-ua/federated/README.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ Now we have the simulated modbus battery we can move on to reading our battery u
...
```
Comment (AIM289) [inline Servers/opc-ua/federated/README.md]: as above

**Review by AIM289 — CHANGES_REQUESTED** [review]

File: Servers/opc-ua/federated/README.md (hunk 1 above)
Comment (JonathanWhiteUK) [reply Servers/opc-ua/federated/README.md]: Done

File: Servers/opc-ua/federated/README.md (hunk 2 above)
Comment (JonathanWhiteUK) [reply Servers/opc-ua/federated/README.md]: Done

**Review by AIM289 — APPROVED** [review]
LGTM

**Pull request merged**